src/main/java/fi/tkgwf/ruuvi/config/Config.java

```diff
@@ -265,4 +276,8 @@ public static boolean isAllowedMAC(String mac) {
     public static String getTagName(String mac) {
         return TAG_NAMES.get(mac);
     }
+
+    public static long currentTimeMillis() {
```

Comment:
Well, I agree it's more Config..ly(?) to return a timestamp provider than a timestamp directly, so I've changed that. 

Testing time-based features has always been a real pain in Java. In the JodaTime library you could just set the system time to a fixed instant without messing with some ugly clock objects or anything. Too bad they didn't copy that feature into Java 8. I don't know why anyone would want to use a `Clock` unless it's for testing. 

I have kept the reflection code for now, I don't think it's too bad to use it for testing time-based features, especially when there's no actual production use case for having a `setTimestampProvider` method. In an integration test I'd like the entire application to use the same time, so that's why I don't like mocking the time of just some single class (even if that's *currently* the only place where time is used).